### ISSUE TITLE
Feature request - HTML & CSS3 Splash Screen Plugin for Ionic 4

### ISSUE THREAD

**Issue by jamesharvey2** [ionic-team/ionic#13704]
Hello Ionic team,

I requested HTML & CSS splash screen feature for Ionic 4 few weeks ago. 
I'd like to share you guys with some examples of amazing HTML & CSS loading screens which can be implemented on Ionic after you guys make it happen: 

https://codepen.io/janrubio/pen/VaVGYr?limit=all&page=4&q=loading+screen

https://codepen.io/kenchen/pen/tgBiE?q=loading+screen&limit=all&type=type-pens

https://codepen.io/alexiscolin/pen/mJQjrr?limit=all&page=3&q=loading+screen

https://codepen.io/mattstvartak/pen/zrQadM

https://codepen.io/niklakhani/pen/cqbiy?limit=all&page=3&q=loading+screen

https://codepen.io/brycesnyder/pen/doLryd?limit=all&page=3&q=loading+screen

https://codepen.io/mathdotrandom/pen/ompBE?limit=all&page=2&q=loading+screen

https://codepen.io/jasonhibbs/pen/maskL?limit=all&page=2&q=loading+screen

https://codepen.io/luka-butina/pen/ZbeQzd?q=loading+screen&limit=all&type=type-pens

https://codepen.io/adamjld/pen/MYaqog?limit=all&page=2&q=loading+screen

Yes, all of these are completely CSS3 and HTML5! 
Imagine all Ionic apps can display an HTML page with CSS3 styles as the sole splash screen! 
it will make a futuristic step forward for app design and make Ionic apps look different from other apps. CSS3 animations can do amazing things. Its possibility is only limited by developers' imagination..
Honestly, I don't understand why all developers are still forced to use a static image file as a splash screen in 2017. An HTML file and CSS3 file combined would be only a few kilobytes and will not take much RAM or CPU at all. 

Current Ionic app is riddled with all kinds of white splash screen problems, gray blank screen problems, loading time problems.. We need an HTML5 & CSS3 splash screen which doesn't get interfered with cordova's gray blank screen & white blank screen.

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us! I am closing this issue because it is a duplicate of Issue #13630, which has been moved into the feature-requests repo already.  Thank you for using Ionic

**Comment by jamesharvey2** [comment]
Thanks, just wanted to showcase a few examples of CSS3 & HTML5 & SVG loading screens. Hope you guys make it happen!

**Comment by Devel0per95** [comment]
i would live too see this feature 😄

**Comment by jamesharvey2** [comment]
Yes, this will make Ionic app look better than other apps which are not using HTML5 & CSS3 as interface. 
There are a plenty of HTML5 & CSS3 animation examples available all over the internet... If Ionic 4 ships with HTML5 loading screen, it'll be easy for all Ionic developers to use them in their app. 

And simple HTML5 & CSS3 files combined will still not be too bigger than a high resolution jpg file.  I think HTML5 is a better candidate for loading screen.

**Comment by jamesharvey2** [comment]
**Benefits of HTML5 & CSS3 splash screen:** 

- HTML5 & CSS3 are responsive. You need to make it only once to use for all kinds of screen sizes.  
- HTML5 & CSS3 file combined are smaller than the sum of image files for different screens. It saves kilobytes. 
- It can be animated and look more engaging. 
- There are many examples of CSS3 loading screens all over the internet.
- Users will no longer need to worry about getting errors with splash screen image file dimensions.
